osu.Game/Storyboards/StoryboardSprite.cs

```diff
@@ -61,8 +62,8 @@ public double EarliestTransformTime
             {
                 // If we got to this point, either no alpha commands were present, or the earliest had a non-zero start value.
                 // The sprite's StartTime will be determined by the earliest command, regardless of type.
-                double earliestStartTime = TimelineGroup.StartTime;
-                foreach (var l in loops)
+                double earliestStartTime = Commands.StartTime;
+                foreach (var l in loopingGroups)
                     earliestStartTime = Math.Min(earliestStartTime, l.StartTime);
```

Comment:
Previously `l.StartTime` referred to the start time of the loop command offset from the start time of the loop group, so the loop group start time was supposed to be added to `l.StartTime` in here for `earliestStartTime` to correctly return the start time of the command...

The behaviour is now different in this PR because `l.StartTime` correctly refers to the absolute start time of the command, making this code more correct.